globalize-strings/globalize.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+
+function cloneRepoCopyGit(){
+    echo "---- START ----"
+    fileName=$1;
+    regex="/(.*)"
+    [[ $fileName =~ $regex ]]
+    repo=${BASH_REMATCH[1]}
+    gitRepo=https://github.com/${fileName}.git
+    echo "Cloning ${gitRepo}"
+    git clone ${gitRepo}
```

Comment:
at this point, the script would ask the user for creds since we're using `https:github.com` right?

Reply:
actually i got something set up in my environment that it doesn't ask me.  
as @kjdelisle mentioned, ssh might be a better choice.